stories/patterns/filters/add-filter/examples/docs/add-filter.tsx

```diff
@@ -0,0 +1,309 @@
+import React from 'react';
+import AddFilterPattern from '@semcore/add-filter-pattern';
+import Select from '@semcore/select';
+import Button from '@semcore/button';
+import { Flex } from '@semcore/flex-box';
+
+import SearchM from '@semcore/icon/Search/m';
+import { ButtonLink } from '@semcore/button';
+import CloseM from '@semcore/icon/Close/m';
+import { Hint } from '@semcore/tooltip';
+import { Text } from '@semcore/typography';
+import Radio, { RadioGroup } from '@semcore/radio';
+import Textarea from '@semcore/textarea';
+import type { AddFilterPatternDropdownProps } from '@semcore/add-filter-pattern';
+import { verify } from 'crypto';
+
+const selectOptions = [
+  { value: 'Option 1', children: 'Option 1' },
+  { value: 'Option 2', children: 'Option 2' },
+];
+
+type SearchFilterInputProps = {
+  placeholder: string;
+  onChange: (v: string) => void;
+  value: string;
+  onClear: () => void;
+};
+
+const SearchFilterInput = ({ value, onClear, onChange, placeholder }: SearchFilterInputProps) => {
+  return (
+    <AddFilterPattern.Search.Input>
+      <AddFilterPattern.Search.Input.Addon>
+        <SearchM />
+      </AddFilterPattern.Search.Input.Addon>
+      <AddFilterPattern.Search.Input.Value
+        value={value}
+        onChange={onChange}
+        w={110}
+        placeholder={placeholder}
+      />
+      {Boolean(value) && (
+        <AddFilterPattern.Search.Input.Addon>
+          <AddFilterPattern.Search.Input.CloseHint
+            tag={ButtonLink}
+            use='secondary'
+            addonLeft={CloseM}
+            title='Clear'
+            onClick={onClear}
+          />
+        </AddFilterPattern.Search.Input.Addon>
```

Comment:
look at the example here please - https://developer.semrush.com/intergalactic/components/input/input-code#input-with-clearing-ability

Reply:
My example is exactly the same as in the link